docs/network-map/infrastructure-view.md

```diff
@@ -0,0 +1,90 @@
+title: Infrastructure View
+description: Navigate your Kubernetes clusters and bare-metal hosts with drill-down capabilities.
+
+The Infrastructure View shows your physical and virtual infrastructure in a hierarchical layout. For Kubernetes environments, you can drill down from clusters to nodes to pods to containers. For bare-metal hosts, you see your servers and what's running on them.
```

Comment:
Can't get to processes from containers? (A container can have more than one process)